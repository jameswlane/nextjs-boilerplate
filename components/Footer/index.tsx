import React, { ReactElement } from 'react'

const Footer = (): ReactElement => {
  return (
    <footer className="footer">
      <div className="container">
        <span className="text-muted">Footer content here.</span>
      </div>
    </footer>
  )
}

export default Footer
