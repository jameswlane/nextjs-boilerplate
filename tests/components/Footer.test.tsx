import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import Footer from '../../components/Footer'

describe('Component => Footer', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Footer />, {})
    const firstRender = asFragment()

    expect(firstRender).toMatchSnapshot()
  })
})
