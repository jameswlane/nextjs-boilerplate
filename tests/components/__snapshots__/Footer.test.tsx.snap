// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component => Footer matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      <span
        class="text-muted"
      >
        Footer content here.
      </span>
    </div>
  </footer>
</DocumentFragment>
`;
